src/core/models/directanswer.js

```diff
@@ -7,22 +7,35 @@ export default class DirectAnswer {
   }
 
   /**
-   * Create a DirectAnswer model from the given server data and formatters
-   * @param {Object} response The server direct answer
-   * @param {Object.<string, function>} formatters The formatters to apply to this direct answer
+   * Constructs an SDK DirectAnswer from an answers-core DirectAnswer
+   *
+   * @param {DirectAnswer} directAnswer from answers-core
+   * @returns {DirectAnswer}
    */
-  static from (response, formatters) {
-    const data = { ...response };
-    const { answer, relatedItem } = data;
-
-    if (answer && formatters[answer.fieldApiName]) {
-      answer.value = formatters[answer.fieldApiName](
-        answer.value,
-        relatedItem.data.fieldValues,
-        relatedItem.verticalConfigId,
-        true);
+  static fromCore (directAnswer) {
```

Comment:
Do we need to apply `formatters` here as well?

Reply:
ya, i'll just go ahead and include the formatters in this PR